docs/storage.md

```diff
@@ -136,7 +136,7 @@ The concept is simple: tell XCP-ng which disk or partition you want to use, and
 As XCP-ng will handle everything for you, be aware that the device or partition will be formatted.
 
 * Don't create a SR over a device or partition that contains important data.
-* If you want to attach an existing SR to your pool, don't create a new local SR over it, else your virtual disks will be deleted. Instead, use the `xe sr-attach` command.
+* If you want to attach an existing SR to your pool, don't create a new local SR over it, else your virtual disks will be deleted. Instead, use the `xe sr-introduce` command. Further explanation can be found in the official Xen/Citrix documentation: https://support.citrix.com/article/CTX121896/how-to-introduce-a-local-storage-repository-in-xenserver
```

Comment:
It was more correct when you had written "Citrix documentation", rather than "Xen/Citrix documentation", because Xen and XenServer are two different things. 

Note: as the entity was recently renamed back to XenServer, you could also write "the official XenServer documentation".

Reply:
I think it should be ok now. I guess next time we are faster if you fix it yourself than guiding a noob through the github process =) but thanks for your help, really appreciated!

Reply:
Well, we always try to empower users if possible. 

I still see two commits instead of just one, so I'll use the Squash and merge button to make them just one. Thanks for your contribution!

Reply:
Thanks for your patience =)